packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2015,18 +2113,18 @@ export class ContainerRuntime
 	 * Parse an op's type and actual content from given serialized content
 	 * ! Note: this format needs to be in-line with what is set in the "ContainerRuntime.submit(...)" method
 	 */
-	private parseOpContent(serializedContent?: string): ContainerRuntimeMessage {
-		assert(serializedContent !== undefined, 0x6d5 /* content must be defined */);
-		const { type, contents, compatDetails }: ContainerRuntimeMessage =
-			JSON.parse(serializedContent);
-		assert(type !== undefined, 0x6d6 /* incorrect op content format */);
-		return { type, contents, compatDetails };
+	private parseLocalOpContent(serializedContents?: string): LocalContainerRuntimeMessage {
+		assert(serializedContents !== undefined, 0x6d5 /* content must be defined */);
+		const message: LocalContainerRuntimeMessage = JSON.parse(serializedContents);
+		assert(message.type !== undefined, 0x6d6 /* incorrect op content format */);
+		return message;
 	}
 
-	private async applyStashedOp(op: string): Promise<unknown> {
+	private async applyStashedOp(serializedOpContent: string): Promise<unknown> {
 		// Need to parse from string for back-compat
-		const { type, contents, compatDetails } = this.parseOpContent(op);
-		switch (type) {
+		const opContents = this.parseLocalOpContent(serializedOpContent);
```

Comment:
For follow-up: I looked around the code a bit and in the cases I checked, `serializedOpContent` appears like it'll be serialized `Outbound` not `Local` message contents.

cc @jason-ha

Reply:
Actually I think we can make this change now, updating `parseLocalOpContent` to be `parseOutboundOpContent` with the corresponding type.

I'm trying to remember how we concluded this should be `Local`.  Maybe the ID Compressor handling was incorrect and we worked back from there...?

Reply:
Right - the IdAllocation case expects there to be stashed state on the op here. I would expect a stashed op that's being applied to be "local" not "outgoing". I guess the problem is that the op _was_ outgoing when it was sent, but now it is local? So, does the distinction between local and outgoing even make sense?

Reply:
Let's discuss after you merge this, as-is more closely matches the code being touched, even if I anticipate it will clash elsewhere in the next wave of changes.